### PR TITLE
support globs in --what<weak_dep> (RhBug:1303311)

### PR DESCRIPTION
**Pull request by MichaelMraka**

### CODE REVIEW

File: plugins/repoquery.py [hunk 1]
```diff
@@ -226,11 +226,9 @@ class RepoQueryCommand(dnf.cli.Command):
...
     def by_dep(sack, pattern, query, dep):
-        try:
-            reldep = hawkey.Reldep(sack, pattern)
-        except hawkey.ValueException:
-            return query.filter(empty=True)
-        kwarg = {dep: reldep}
+        if dnf.util.is_glob_pattern(pattern):
+            dep += '__glob'
+        kwarg = {dep: pattern}
         return query.filter(**kwarg)
```
Comment (mluscon) [inline plugins/repoquery.py]: `- import hawkey` then. BTW, isn't hawkey.Reldep faster than filter?

Comment (MichaelMraka) [inline plugins/repoquery.py]: Reldep doesn't work with globs.

Comment (mluscon) [inline plugins/repoquery.py]: Sure, but you can use it if the pattern is not glob.

Comment (MichaelMraka) [inline plugins/repoquery.py]: > BTW, isn't hawkey.Reldep faster than filter?

It's the same. (Exactly same string to reldep conversion is the first thing which happen in the filter.)

**Comment by mluscon** [comment]
Ok, LGTM now.

**Comment by mluscon** [comment]
Merged

**Pull request closed without merging**